src/main/java/org/truffleruby/core/exception/CoreExceptions.java

```diff
@@ -67,6 +66,15 @@ public void showExceptionIfDebug(DynamicObject rubyClass, Object message, Backtr
         }
     }
 
+    @TruffleBoundary
+    public void showExceptionIfDebug(DynamicObject rubyException, Backtrace backtrace) {
+        if (context.getCoreLibrary().getDebug() == Boolean.TRUE) {
+            final DynamicObject rubyClass = Layouts.BASIC_OBJECT.getLogicalClass(rubyException);
+            final Object message = context.send(rubyException, "to_s");
+            showExceptionIfDebug(rubyClass, message, backtrace);
+        }
+    }
```

Comment:
Could you move it one method above? So both helpers are before the "real logic with all args method".

Reply:
I've moved it, but GH is still showing it for some reason.